### PR TITLE
FEX-171 Enable external data sources into activity feed

### PR DESCRIPTION
**Pull request by yozzo**
## Description of change

This PR enables users to see in the activity feed events which come from external sources as Companies House Incorporation, Accounts Due and HMRC Exporters.

The external sources go through a matching process and then are feed into the Activity Stream from where we get them from.

## Test instructions

In the Company Activity Feed Tab there should be visible cards that carry the 'Companies House' and HMRC labels.

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to develop?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2170,6 +2170,13 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.8.0.tgz#f0e003d9ca9e7f59c7a508945d7b2ef9a04a542f"
   integrity sha512-ReZxvNHIOv88FlT7rxcXIIC0fPt4KZqZbOlivyWtXLt8ESx84zd3kMC6iK5jVeS2qt+g7ftS7ye4fi06X5rtRQ==
 
+axios-mock-adapter@^1.17.0:
```
Comment (yeahfro) [inline yarn.lock]: 😢 this will be removed in the next release

**Comment by yeahfro** [comment]
Would be nice to get more description on PR.

**Comment by yeahfro** [comment]
Small thing I notice is that the commit message is prefixed with a code whereas the agreed standard would be a present tense verb.

**Comment by yozzo** [comment]
> Small thing I notice is that the commit message is prefixed with a code whereas the agreed standard would be a present tense verb.

We, the yellow team, need the Jira code prefix in the message for it to be visible in the ticket.

**Comment by yeahfro** [comment]
> We, the yellow team, need the Jira code prefix in the message for it to be visible in the ticket.

Does it need to be at the beginning? Can it be at the end or in the description?

**Comment by yozzo** [comment]
> > We, the yellow team, need the Jira code prefix in the message for it to be visible in the ticket.
> 
> Does it need to be at the beginning? Can it be at the end or in the description?

yes, it has to be at the beginning as per their documentation: https://confluence.atlassian.com/adminjiracloud/connect-jira-cloud-to-github-814188429.html   (paragraph at the bottom of the page).

Does this affect any automation, or it shows up in the release notes?

**Pull request merged**